addons/test_discuss_full/tests/test_performance.py

```diff
@@ -142,7 +143,7 @@ class TestDiscussFullPerformance(HttpCase, MailCommon):
     #       - search user (_author_to_store)
     #       - fetch user (_author_to_store)
     #       - _compute_rating_stats
-    _query_count_discuss_channels = 59
+    _query_count_discuss_channels = 60
```

Comment:
I'm surprised we have an increase here, I guess it is a caching issue depending on order of fetches?